modules/appeals_api/app/models/appeals_api/notice_of_disagreement.rb

```diff
@@ -326,7 +331,10 @@ def birth_date
     def birth_date_is_in_the_past
       return unless birth_date
 
-      add_error("Veteran birth date isn't in the past: #{birth_date}") unless self.class.past? birth_date
+      unless self.class.past? birth_date
+        add_error("Veteran birth date isn't in the past: #{birth_date}",
+                  title: "Birth date isn't in the past", pointer: '/', code: 144)
```

Comment:
I think we should add the `meta` key here showing that the `X-VA-Birth-Date` is where the validation failed. Maybe an `invalid_fields` key with the array being `['X-VA-Birth-Date']`? Whatever it is we use for code 144's meta details, anyway 😅